src/components/HomePage/Forms/MovieForm.tsx

```diff
@@ -1,76 +1,119 @@
-import React, {FC, FormEvent, useState} from 'react';
-import { useDispatch } from 'react-redux';
-import { Form } from 'reactstrap';
-import { addMovieCreator, updateMovieCreator } from '../../../redux/actions/actionCreators/actionCreators';
+import React, { FC } from 'react';
+import { Formik, Form } from 'formik';
+import { useDispatch } from "react-redux";
 import { MovieInfoDetails } from '../../../types';
 import { ResetButton, SubmitButton } from '../../shared';
-import './Form.scss';
 import { InputWithLabel } from './InputWithLabel';
 import { SelectInputGenre } from './SelectInputGenre';
+import {
+    addMovieCreator, toggleCurrentGenresCreator,
+    updateMovieCreator
+} from "../../../redux/actions/actionCreators/actionCreators";
+import './Form.scss';
```

Comment:
a bit inconsistent with quotes... Let's use single or double for imports (and add this role to the linter)

Reply:
Thank you! Added an additional rule to eslint configs and fixed in commit 144e7682797a8b327df9c556453f21aad1d10993